pkg/manager/manager.go

```diff
@@ -82,7 +82,6 @@ func New(ctx context.Context, opts Options) (Manager, error) {
 
 	// Build the controller manager context.
 	controllerManagerContext := &capvcontext.ControllerManagerContext{
```

Comment:
Should we rename the function inside the Manager interface to `GetControllerManagerContext` instead of `GetContext`?

Reply:
Sure, it would make the code easier to read